src/lib/storage/disks_list.rb

```diff
@@ -0,0 +1,58 @@
+#
+# encoding: utf-8
+
+# Copyright (c) [2016] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "storage"
+require "storage/devices_list"
+require "storage/partitions_list"
+require "storage/filesystems_list"
+#require "storage/free_disk_spaces_list"
+require "storage/refinements/disk"
+
+module Yast
+  module Storage
+    class DisksList < DevicesList
```

Comment:
I dislike class names in plural. It's easy to confuse them with the singular class (Disk vs Disks) and they challenge you to think in unnatural ways like "create a devices" or "compare two devices..es..ermm, wait, what's the plural of 'devices'?". I prefer to "create a list of devices" and "compare two list of devices".